### ISSUE TITLE
fn2yara segfaults

### ISSUE THREAD

**Issue by sei-eschwartz** [cmu-sei/pharos#264]
```
Program received signal SIGSEGV, Segmentation fault.
(anonymous namespace)::FnToYaraAnalyzer::visit (this=0x7fffffffca50, fd=0x49f3af8) at /home/mwd/git/pharos/master/tools/fn2yara/fn2yara.cpp:593
warning: Source file is more recent than executable.
593                 wildcard[off/8] = true;
Missing separate debuginfos, use: debuginfo-install bzip2-libs-1.0.6-13.el7.x86_64 glibc-2.17-326.el7_9.x86_64 gmp-6.0.0-15.el7.x86_64 libgcc-4.8.5-44.el7.x86_64 libgcrypt-1.5.3-14.el7.x86_64 libgpg-error-1.12-3.el7.x86_64 libicu-50.2-4.el7_7.x86_64 libstdc++-4.8.5-44.el7.x86_64 ncurses-libs-5.9-14.20130511.el7_4.x86_64 readline-6.2-11.el7.x86_64 sqlite-3.7.17-8.el7_7.1.x86_64 xz-libs-5.2.2-2.el7_9.x86_64 zlib-1.2.7-21.el7_9.x86_64
(gdb) bt
#0  (anonymous namespace)::FnToYaraAnalyzer::visit (this=0x7fffffffca50, fd=0x49f3af8) at /home/mwd/git/pharos/master/tools/fn2yara/fn2yara.cpp:593
#1  0x00007ffff7552ca7 in pharos::BottomUpAnalyzer::<lambda(size_t, pharos::FunctionDescriptor*)>::operator()(size_t, pharos::FunctionDescriptor *) const (
    __closure=__closure@entry=0x7fffffffc6c0, fd=0x49f3af8) at /home/mwd/git/pharos/master/libpharos/bua.cpp:119
#2  0x00007ffff75549cb in pharos::BottomUpAnalyzer::analyze (this=this@entry=0x7fffffffca50) at /home/mwd/git/pharos/master/libpharos/bua.cpp:150
#3  0x000000000041bc43 in (anonymous namespace)::fn2yara_main (argc=3, argv=<optimized out>) at /home/mwd/git/pharos/master/tools/fn2yara/fn2yara.cpp:658
#4  0x00007ffff765d6a7 in pharos::pharos_main (glog_name="FN2Y", fn=fn@entry=0x41a30b <(anonymous namespace)::fn2yara_main(int, char**)>, argc=argc@entry=3, argv=argv@entry=0x7fffffffd348, 
    logging_fileno=logging_fileno@entry=1) at /home/mwd/git/pharos/master/libpharos/options.cpp:842
#5  0x0000000000419dee in main (argc=3, argv=0x7fffffffd348) at /home/mwd/git/pharos/master/tools/fn2yara/fn2yara.cpp:667
(gdb) 
(gdb) p wildcard
$1 = std::vector<bool> of length 2, capacity 64 = {1, 1}
(gdb) p off/8
$2 = 5588352
```

[exe.zip](https://github.com/cmu-sei/pharos/files/14807912/exe.zip)

**Comment by sei-mwd** [comment]
Should be fixed with 2089ae0527f4f1b3908c10eaea3ab375e7099ee7.

**Comment by sei-mwd** [comment]
Meant to close.
